CONTRIBUTING.md

```diff
@@ -123,9 +123,9 @@ To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/sys
 
 ## Debug
 
-To debug the plugin using IntelliJ you need to add a `Run Configuration`, to do so go to `Run > Edit Configurations...`.
-Use the plus button in the right upper corner to add a new configuration and choose Gradle from the drop-down list.
-In the `Run` section input `runIde`. <br/>
+To debug the plugin using IntelliJ go to `Run > Edit Configurations...` and create a new configuration for Gradle
+![image](https://user-images.githubusercontent.com/61919032/181751590-cc21dd74-f481-4b86-b0d9-b99836669bfd.png)
```

Comment:
Whoops I wouldn't link to an image from outside the repo, it's not clear how long github is gonna keep it around... pls just include the image in the repo and use a relative path, to keep everything self-contained